applying-stpa/l1_reqs.txt

```diff
@@ -0,0 +1,228 @@
+REQ-5: The OpenAPS rig should emit a warning if it's going out of range of the insulin pump
+REQ-6: If the insulin pump goes out of range of OpenAPS, it should fall back to normal operation
+REQ-7: OpenAPS should have watchdog timer or heartbeat communications between systems to detect if insulin pump is working properly and if not warn the user.
+REQ-8: If the Insulin Pump is on low battery, OpenAPS should stop sending it commands and let it fall back to normal operation
+REQ-9: The OpenAPS algorithm must set a temporary basal dosing recommendation if the insulin dosage is not optimal and it has been over X minutes since the last temporary basal recommendation
+REQ-10: OpenAPS algorithm should be tested previously, validated in realtime for accuracy and designed securely
+REQ-11: If OpenAPS is having problems communicating with other devices in the loop, OpenAPS should indicate this and fallback to the human programmed basal rates
+REQ-12: OpenAPS shall warn the human actor if it is no longer connected to and/or no longer receiving information from Nightscout.
+REQ-13: There should be checks for wireless interference every X minutes, and OpenAPS should send a warning to the user if there is a positive result
+REQ-14: OpenAPS plausibility check shall include examination of the raw data from the insulin pump to ensure the data is in its expected range
+REQ-15: OpenAPS must inform the human actor if the insulin pump is set to use the wrong units
+REQ-16: If the OpenAPS rig components are not connected together properly there must be an alarm, warning or error
+REQ-17: The OpenAPS rig components must connect together securely
+REQ-18: User manual must specify the correct hardware setup
+REQ-19: Software must check that all required OpenAPS dependencies are present
+REQ-20: Hardware connections need to be set up to match the configuration the human entered to the openAPS configuration.
+REQ-21: Communication between components must have an integrity check
+REQ-22: Must have watchdog timer or heartbeat communications between systems to detect if insulin pump is working properly and if not warn the user.
+REQ-23: OpenAPS must alert human actor if the insulin pump has stopped giving feedback
+REQ-24: OpenAPS should be able to interpret error signals from insulin pump and convey to the human actor
+REQ-25: The OpenAPS rig must have as long a battery life as possible
+REQ-26: The OpenAPS rig must alert the user when on low power
+REQ-27: The OpenAPS rig must have a safe exit process
+REQ-28: Docs must have guidelines on avoiding a low power supply
+REQ-29: If OpenAPS is having problems communicating with other devies in the loop, OpenAPS should indicate this and fallback to the human programmed basal rates
+REQ-30: OpenAPS wireless network signals must have a radio frequency between X MegaHertz and Y MegaHertz
+REQ-31: OpenAPS documentation must direct the human actor to easily use the right dependencies, perhaps via an automated script
```

Comment:
let's be specific about which dependencies.   I'm thinking this is software dependencies for the OpenAPS system, but ??